README.md

```diff
@@ -59,22 +122,31 @@ The `adminPasswordOrKey` parameter in this example command uses an existing SSH
 When the ARM template completes successfully you will have an AKS cluster in the provided resource group.
 
 ```sh
-az aks list --resource-group <resource-group>
+az aks list --resource-group $OAM_TUTORIAL_RESOURCE_GROUP_NAME
 ```
 
-Find the cluster name in the command output
+Export the AKS cluster name to a environment variable:
+
+```sh
+export OAM_TUTORIAL_AKS_CLUSTER_NAME=$(az aks list --resource-group $OAM_TUTORIAL_RESOURCE_GROUP_NAME | jq -r '.[0].name')
```

Comment:
You don't need jq for this, az cli has filtering built in https://docs.microsoft.com/en-us/cli/azure/query-azure-cli?view=azure-cli-latest#get-a-single-value

Reply:
Done

Reply:
Ex:
```
az aks list --query "[0].name" --output tsv
```